pkg/minikube/node/start.go

```diff
@@ -401,14 +414,30 @@ func configureRuntimes(runner cruntime.CommandRunner, cc config.ClusterConfig, k
 		}
 	}
 
-	if kv.GTE(semver.MustParse("1.24.0-alpha.2")) {
-		if err := cruntime.ConfigureNetworkPlugin(cr, runner, cc.KubernetesConfig.NetworkPlugin); err != nil {
-			exit.Error(reason.RuntimeEnable, "Failed to configure network plugin", err)
+	inUserNamespace := strings.Contains(cc.KubernetesConfig.FeatureGates, "KubeletInUserNamespace=true")
+	// for docker container runtime: ensure containerd is properly configured by calling Enable(), as docker could be bound to containerd
+	// it will also "soft" start containerd, but it will not disable others; docker will disable containerd if not used in the next step
+	if co.Type == "docker" {
+		containerd, err := cruntime.New(cruntime.Config{
+			Type:              "containerd",
+			Socket:            "", // use default
+			Runner:            co.Runner,
+			ImageRepository:   co.ImageRepository,
+			KubernetesVersion: co.KubernetesVersion,
+			InsecureRegistry:  co.InsecureRegistry})
+		if err == nil {
+			err = containerd.Enable(false, cgroupDriver(cc), inUserNamespace) // do not disableOthers, as it's not primary cr
+		}
+		if err != nil {
+			klog.Warningf("cannot ensure containerd is configured properly and reloaded for docker - cluster might be unstable: %v", err)
 		}
 	}
 
-	inUserNamespace := strings.Contains(cc.KubernetesConfig.FeatureGates, "KubeletInUserNamespace=true")
-	err = cr.Enable(disableOthers, forceSystemd(), inUserNamespace)
+	disableOthers := true
+	if driver.BareMetal(cc.Driver) {
+		disableOthers = false
+	}
```

Comment:
```suggestion
	disableOthers := !driver.BareMetal(cc.Driver)
```
nit: I find it cleaner and easier to read "disableOthers if not bare metal"

Reply:
agree